libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/operations/CompiledSelectionsBuilder.kt

```diff
@@ -125,7 +125,7 @@ internal class CompiledSelectionsBuilder(
   private fun IrArgument.codeBlock(): CodeBlock {
     val argumentBuilder = CodeBlock.builder()
     argumentBuilder.add(
-        "%T(%T.%L)",
+        "%T(%T.%N)",
         KotlinSymbols.CompiledArgument,
         context.resolver.resolveArgumentDefinition(definitionId),
         definitionPropertyName,
```

Comment:
Same here, not really needed because there is already `__` in front of `definitionPropertyName` but still feels more correct